Readme.md

```diff
@@ -46,20 +46,20 @@ sudo apt-get install build-essential clang libssl-dev pkg-config libreadline-dev
 curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
 source $HOME/.cargo/env
 ```
-1) Install our fork of the PGX framework
+1) Install the PGX framework
 ```bash
-cargo install --git https://github.com/JLockerman/pgx.git --branch timescale cargo-pgx
+cargo install cargo-pgx
 ```
-1) Initialize the PGX framework using the PostgreSQL 13 installation
+1) Initialize the PGX framework using the PostgreSQL 14 installation
 ```bash
-cargo pgx init --pg13=/usr/lib/postgresql/13/bin/pg_config
+cargo pgx init --pg14=/usr/lib/postgresql/14/bin/pg_config
```

Comment:
if we are switching to pg14 here we should change the version above 13->14 too (line 42,33,29)

Reply:
lines 30-34 must be replaced with 
```
sudo sh -c "echo 'deb [signed-by=/usr/share/keyrings/timescale.keyring] https://packagecloud.io/timescale/timescaledb/ubuntu/ $(lsb_release -c -s) main' > /etc/apt/sources.list.d/timescaledb.list"
wget --quiet -O - https://packagecloud.io/timescale/timescaledb/gpgkey | sudo gpg --dearmor -o /usr/share/keyrings/timescale.keyring
sudo apt-get update
sudo apt install timescaledb-2-postgresql-13
```

Reply:
👍 I have adjusted these steps and run through them myself from beginning to end. The only thing that doesn't work _right now_ is `wget --quiet -o "promscale_extension.zip" "https://github.com/timescale/promscale_extension/archive/refs/tags/0.3.0.zip"` because the tag doesn't exist yet.